### PR TITLE
Article: Bitwise Operators

### PR DESCRIPTION
**Pull request by ds-249**

### CODE REVIEW

File: Bitwise-Operators.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+##Bitwise Operators
+
+<< , >> , & , | , ~, and ^ are the bitwise operators which operates on one or more bit patterns or binary numerals at the level of their individual bits.
```
Comment (atjonathan) [inline Bitwise-Operators.md]: ~~<< , >> , & , | , ~, and ^~~ `<<` , `>>` , `&` , `|` , `~`, and `^`

~~operates~~ operate

File: Bitwise-Operators.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+##Bitwise Operators
```
Comment (atjonathan) [inline Bitwise-Operators.md]: ~~##Bitwise Operators~~ # Bitwise Operators

File: Bitwise-Operators.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+<< , >> , & , | , ~, and ^ are the bitwise operators which operates on one or more bit patterns or binary numerals at the level of their individual bits.
+
+x << y  -> Left shift operator
```
Comment (atjonathan) [inline Bitwise-Operators.md]: ~~x << y  -> Left shift operator~~ ##  `x << y  ->` Left Shift Operator

File: Bitwise-Operators.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+Returns x with the bits shifted to the left by y places (and new bits on the right-hand-side are zeros). This is the same as multiplying x by 2**y.
+
+x >> y  -> Right shift operator
```
Comment (atjonathan) [inline Bitwise-Operators.md]: ~~x >> y  -> Right shift operator~~ ##  `x >> y  ->` Right Shift Operator

**Comment by ds-249** [comment]
@atjonathan should I be making those changes you've suggested? and request a new PR?

File: Bitwise-Operators.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+#Bitwise Operators
```
Comment (atjonathan) [inline Bitwise-Operators.md]: ~~#Bitwise Operators~~ # Bitwise Operators

**Comment by atjonathan** [comment]
### **LGTM :+1:**

Second opinion wanted from @FreeCodeCamp/wiki-mods

**Comment by Rafase282** [comment]
Are they the same in all other languages?

Looks good to me but it would be nice to know if this is in general or for specific languages.

**Comment by ds-249** [comment]
They're not necessarily same in all languages. But they are the same in the popular languages like Java, Python, Ruby, C, C++ etc.

**Comment by alayek** [comment]
@ds-249 actually, it's not. While what you said, is mostly true, there are some nuances.

For example, in some languages, there are unsigned right shift and signed right shift. If you note assembly language instruction set, there are distinction between these two because the sign bit can be replaced by same bit, or by a zero.

Maybe you can mention that in the article. If what I said didn't make any sense, you might wanna consult [this](https://www.cs.umd.edu/class/sum2003/cmsc311/Notes/BitOp/bitshift.html)

**Comment by ds-249** [comment]
@alayek will these changes suffice?

**Comment by abhisekp** [comment]
@ds-249 please add some heading 2 headings.

**Comment by ds-249** [comment]
@abhisekp Done

**Comment by alayek** [comment]
# LGTM :+1:  

Let's :shipit: :package:

**Comment by abhisekp** [comment]
@ds-249 Perfecto :heart_eyes:

**Pull request merged**